src/netaddress.cpp

```diff
@@ -440,12 +445,15 @@ std::vector<unsigned char> CNetAddr::GetGroup(const std::vector<bool> &asmap) co
         nStartByte = 6;
         nBits = 4;
     }
-    // for he.net, use /36 groups
-    else if (GetByte(15) == 0x20 && GetByte(14) == 0x01 && GetByte(13) == 0x04 && GetByte(12) == 0x70)
+    else if (IsHeNet())
+    {
+        // for he.net, use /36 groups
         nBits = 36;
-    // for the rest of the IPv6 network, use /32 groups
-    else
+    }
+    else {
+        // for the rest of the IPv6 network, use /32 groups
         nBits = 32;
+    }
```

Comment:
Should fix brackets here (specifically this range, to match upstream commit

Reply:
Headers are eventually normalised [here](https://github.com/dashpay/dash/pull/4028/commits/a343f996c1a36fd0f688a0fb0644f107259e826b#diff-1ce9cdfdcb1a8474c626ebd0d6eb261d2a91d4c6ac1bbed8feb57a94c61b5701L409-R465)